content/posts/2021/release-0.8.0.md

```diff
@@ -0,0 +1,362 @@
+---
+title: "REANA 0.8.0 is released"
+date: 2021-11-30T10:19:06+01:00
+tags:
+  - release
+  - client
+  - cluster
+---
+
+We are glad to announce the release of REANA 0.8.0, a major update
+allowing users to run Snakemake workflows, allowing administrators to
+set and monitor CPU and Disk quotas for users, and more!
+
+<!--more-->
+
+
+## What's new for the users?
+
+### Support for running Snakemake workflows
+
+One of the most notable new features of REANA 0.8 release series is
+the support for running [Snakemake](https://snakemake.github.io/)
+workflows. Snakemake joins CWL and Yadage as another wokflow
```

Comment:
wokflow -> workflow

Reply:
Thanks, all fixed. Forgot to run spell checker :worried: